shoop_tests/core/test_module_interface.py

```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8 -*-
+# This file is part of Shoop.
+#
+# Copyright (c) 2012-2015, Shoop Ltd. All rights reserved.
+#
+# This source code is licensed under the AGPLv3 license found in the
+# LICENSE file in the root directory of this source tree.
+from __future__ import unicode_literals
+import pytest
+
+from shoop.core.models import Supplier
+from shoop.testing.factories import get_default_supplier
+
+
+@pytest.mark.django_db
+def test_module_interface_for_scandinavian_letters(rf):
+    supplier = get_default_supplier()
```

Comment:
This test shouldn't modify the default supplier. (It's fine to create a new supplier for this test -- and if you want, you can even `.delete()` it at the end of this test to clean things up)